src/vmware/CHANGELOG.md

```diff
@@ -1,8 +1,17 @@
 # Release History
 
-## 2.1.0 (TBD)
-- add az vmware private-cloud rotate-vcenter-password
-- add az vmware private-cloud rotate-nsxt-password
+## 3.0.0 (2021-06)
+This version has **breaking changes**:
+- `az vmware datastore create` has been removed. Please use `az vmware datastore netapp-volume create` or `az vmware datastore disk-pool-volume create` instead.
+
+Other changes:
+- update to AVS 2021-06-01 API
+- add `az vmware private-cloud rotate-vcenter-password`
+- add `az vmware private-cloud rotate-nsxt-password`
+- add `az vmware addon` command group
+- add `az vmware addon hcx` command group to manage a HCX addon
+- add `az vmware addon srm` command group to manage a Site Recovery Manager (SRM) addon
+- add `az vmware addon vr` command group to manage a vSphere Replication (VR) addon
```

Comment:
Please capitalize the first letter of those sentences

Reply:
I capitalized them.